dynamics-nav/SETRANGE-Function--Query-.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "SETRANGE Function (Query)"
+description: SETRANGE Function (Query)Sets a filter on a range of values on a column of a query dataset.This topic contains information on the SETRANGE Function (Query) syntax, parameters, remarks, and examples.
```

Comment:
Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.